firmware/lib/Gantry/LimitSwitch.h

```diff
@@ -3,10 +3,15 @@
 
 #include "Arduino.h"
 
+typedef enum {
+    PRESSED = LOW,
+    RELEASED = HIGH
```

Comment:
This is just preference, but we can also skip value assignment and use default enum values:
`typedef enum{PRESSED,RELEASED};`

Reply:
By definition, these have to be equal to LOW and HIGH, so I'd rather explicitly declare that rather than rely on the implicit dependency that PRESSED == 0 and LOW == 0 therefore PRESSED == LOW